lib/numbers.md

```diff
@@ -5,20 +5,83 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Integers (aka Fixnum) are whole numbers. 
+When used in mathematical operations they will always return whole numbers.
+
+    15/2 = 7
+    
+Floats are numbers with a decimal point. Floats great than -1 and less than 1 require a zero in front of the decimal point. 
+When used in mathematical operations they will always return floats.
+
+    15/2.0 = 7.5
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
++ (addition)
++= (combined operator....
+all of the combined operators perform an arithmetic operation on the variable before assigning it a new value
+example:
+    
+    x=1
+    x+=2  (translation: x=x+2)
+    now x=3
+)
+
+- (subtraction)
+-= (combined operator)
+* (multiplication)
+*= (combined operator)
+
+    x=2
+    x*=3 (translation x=x*3)
+    now x=6
+    
+/ (division)
+/= (combined operator)
+** (exponent)
+% (modulus- Divides left # by right # and returns remainder)
+.to_i (converts float to integer)
+.to_f (converts integer to float)
+== (asks if true)
+!= (asks if false)
+> (asks if greater than)
+< (asks if less than)
+>= (asks if greater than or equal to)
+<= (asks if less than or equal to)
+<=> (combined comparison...if both numbers are same it returns 0, if first number is greater than second number it returns 1, and if the opposite is true it returns -1)
+
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+Using the '+' on two numbers will execute the mathematical operation of addition while using it on two strings will combine the strings to form one string.
+
+    1 + 1 = 2
+vs.
+
+    "1" + "1" = "11"
+
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+No, it won't work as a mathematical operation because your computer won't recognize "20" as a number. 
+It is only a collection of characters to your computer. 
+You will need to convert the string to an integer and also possibly a float
+
+x="20"
+x.to_i.to_f/3
```

Comment:
Ng. I think I was falsely assuming you had to convert a string to an integer based on some old problems I'd done like a year ago that I am remembering incorrectly. It makes sense when you put it in terms of a string that contains "20.5". Converting to an integer would remove the ".5", leaving only the "20". If I then converted it to a float it would be "20.0".